services_and_applications/slate/storage.rst

```diff
@@ -220,6 +207,51 @@ An example Persistent Volume Claim that implements snapshots would look similar
        requests:
          storage: 1Gi
 
+We can now create a **VolumeSnapshot**, which will capture the data in the **PersistentVolume** that was provisioned by the named **PersistentVolumeClaim** at the time
+the **VolumeSnapshot** is created, to backup the data. 
+
+.. code-block:: yaml
+
+    apiVersion: snapshot.storage.k8s.io/v1beta1
+    kind: VolumeSnapshot
+    metadata:
+      # Name of VolumeSnapshot
+      name: pvc-snap
+    spec:
+      source:
+        # Name of persistentVolumeClaim to snapshot
+        persistentVolumeClaimName: snapshot-pvc
+      volumeSnapshotClassName: csi-snapclass
```

Comment:
This VolumeSnapshotClass name seems kind of arbitrary from a user perspective. If we don't have any "options" we need to provide trident with this maybe we can name it `netapp-default` and then make it the default storage class with the annotation `snapshot.storage.kubernetes.io/is-default-class: "true"`

Reply:
I added a new VolumeSnapshotClass to the clusters named netapp-default and removed the default annotation on the existing csi-snapclass VolumeSnapshotClass. I left the csi-snapclass in place since there is at least one user on Marble that is using VolumeSnapshots with the csi-snapclass VolumeSnapshotClass.